src/IIIFPresentation/Core/WriteResult.cs

```diff
@@ -48,5 +48,10 @@ public enum WriteResult
     /// <summary>
     ///     Predefined storage limits exceeded
```

Comment:
The contents of the `<summary>` blocks in this file are indented, but this style doesn't seem to be used elsewhere.